deluge/ui/web/server.py

```diff
@@ -435,7 +435,17 @@ def __init__(self):
         self.putChild('gettext.js', GetText())
         self.putChild('flag', Flag())
         self.putChild('icons', LookupResource('Icons', rpath('icons')))
-        self.putChild('images', LookupResource('Images', rpath('images')))
+        images = LookupResource(
+            'Images',
+            rpath('images'),
+            os.path.join(
+                os.path.dirname(os.path.abspath(__file__)),
```

Comment:
The data dir path for ui will not always be at this location so should use `resource_filename` with module name `deluge.ui` and I would also use a separate location to differentiate from web ui images. e.g. `putChild('ui_images', ...`